tasks/configure.yml

```diff
@@ -38,17 +38,7 @@
   register: config_updated
   notify: restart elasticsearch
 
-
-# - name: Install additional plugins
-#   elasticsearch_plugin:
-#     name: "{{ item }}"
-#     state: present
-#   with_items: elasticsearch_plugins
-
-#- name: Copy Default File for Instance
-#  template:
-#    src: elasticsearch.j2
-#    dest: "{{ instance_default_file }}"
-#    mode: 0644
-#    force: yes
-#  notify: restart elasticsearch
+- name: Enable ES service
+  service:
```

Comment:
According to documentation this module needs `state` defined when enabling service. https://docs.ansible.com/ansible/latest/service_module.html#options